homeassistant/components/system_bridge/sensor.py

```diff
@@ -415,6 +716,54 @@ def extra_state_attributes(self) -> dict[str, Any]:
         return attrs
 
 
+class SystemBridgeProcessesCpuLoadSensor(SystemBridgeSensor):
+    """Defines a Processes CPU Load sensor."""
+
+    def __init__(
+        self, coordinator: SystemBridgeDataUpdateCoordinator, index: int
+    ) -> None:
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            f"processes_load_cpu_{index}",
+            f"Load CPU {index}",
+            "mdi:percent",
+            None,
+            PERCENTAGE,
+            False,
+        )
+        self._index = index
+
+    @property
+    def state(self) -> float | None:
+        """Return the state of the sensor."""
+        bridge: Bridge = self.coordinator.data
+        return (
+            round(bridge.processes.load.cpus[self._index].load, 2)
+            if bridge.processes.load.cpus[self._index].load is not None
+            else None
+        )
+
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Return the state attributes of the entity."""
+        bridge: Bridge = self.coordinator.data
+        attrs = {}
+        if bridge.processes.load.cpus[self._index].loadUser is not None:
+            attrs[ATTR_LOAD_USER] = round(
```

Comment:
Why aren't these separate sensors?

Reply:
I might remove them. They don't really accomplish anything apart from splitting up user, system and idle load

Reply:
We could make them separate sensors and disabled by default?